client/web/src/auth/SignInPage.tsx

```diff
@@ -112,11 +112,7 @@ export const SignInPage: React.FunctionComponent<React.PropsWithChildren<SignInP
                 iconLinkTo={props.context.sourcegraphDotComMode ? '/search' : undefined}
                 iconClassName="bg-transparent"
                 lessPadding={true}
-                title={
-                    props.context.sourcegraphDotComMode
-                        ? 'Sign in to Sourcegraph Cloud'
-                        : 'Sign in to Sourcegraph Server'
-                }
+                title={props.context.sourcegraphDotComMode ? 'Sign in to Sourcegraph Cloud' : 'Sign in to Sourcegraph'}
```

Comment:
I think "Sourcegraph Cloud" now refers to Sourcegraph MIs, but can't think of a better name or maybe we don't need to distinguish between dotcom and on-perm copyings at all?
```suggestion
                title={'Sign in to Sourcegraph Cloud'}
```